integration/utils/transactions.go

```diff
@@ -126,6 +130,73 @@ func MakeCollectionRegisterNodeTx(
 	return tx, nil
 }
 
+func MakeStakingCollectionCloseStakeTx(
+	env templates.Environment,
+	stakingAccount *sdk.Account,
+	stakingAccountKeyID int,
+	stakingSigner crypto.Signer,
+	payerAddress sdk.Address,
+	latestBlockID sdk.Identifier,
+	nodeID flow.Identifier,
+	sign bool,
+) (*sdk.Transaction, error) {
+	accountKey := stakingAccount.Keys[stakingAccountKeyID]
+	tx := sdk.NewTransaction().
+		SetScript(templates.GenerateCollectionCloseStake(env)).
+		SetGasLimit(9999).
+		SetReferenceBlockID(latestBlockID).
+		SetProposalKey(stakingAccount.Address, stakingAccountKeyID, accountKey.SequenceNumber).
+		SetPayer(payerAddress).
+		AddAuthorizer(stakingAccount.Address)
+
+	id, _ := cadence.NewString(nodeID.String())
+	err := tx.AddArgument(id)
+	if err != nil {
+		return nil, err
+	}
+
+	err = tx.AddArgument(cadence.NewOptional(nil))
+	if err != nil {
+		return nil, err
+	}
+
+	if sign {
+		err = tx.SignPayload(stakingAccount.Address, stakingAccountKeyID, stakingSigner)
+		if err != nil {
+			return nil, fmt.Errorf("could not sign payload: %w", err)
+		}
+	}
+
+	return tx, nil
+}
+
+// MakeAdminRemoveNodeTx makes the admin remove node transaction
```

Comment:
Could add a bit more detail here like, this is equivalent to the node unstaking and will result in removal at the next epoch boundary.

Reply:
https://github.com/onflow/flow-go/pull/1728/commits/657e37d5bb69949bc8b30a6ca1acf70570779c71